Sources/PackageGraph/DependencyResolver.swift

```diff
@@ -675,7 +727,8 @@ public class DependencyResolver<
     func resolveSubtree(
         _ container: Container,
         subjectTo allConstraints: ConstraintSet,
-        excluding allExclusions: [Identifier: Set<Version>]
+        excluding allExclusions: [Identifier: Set<Version>],
+        containers: Set<Identifier>
```

Comment:
I think this might be required if dependency constraints for this subtree are not in the main loop and some dependency container should be ignored? (Though this is a speculation...I'll find out via tests.)